frontend/src/app/timezone.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { TimezoneService } from './timezone.service';
+
+describe('TimezoneService', () => {
+  let service: TimezoneService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(TimezoneService);
+  });
+
+  it('should be created', () => {
```

Comment:
**question (if-minor):** Do we have documentation in our README on how to run these tests during local development? And are they running now in CI, or is that part of the planning work that we are still doing? If not, would adding the running of these tests to the existing setup be worth doing in this PR?

Reply:
@neilmb Glad you're paying attention to this. I do think this is best done as part of another story. This one is just to connect the apps, but perhaps #289 would make a good home for this (even though they're unit tests).